contrib/envoyproxy/envoy/envoy.go

```diff
@@ -0,0 +1,338 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016 Datadog, Inc.
+
+package envoy
```

Comment:
could you please add an `example_test.go` file showing how this integration is used with some sort of real world example? 🙏

Reply:
Added in 13ceb5e 😄